scripts/clean-db-dump.sql

```diff
@@ -126,18 +130,19 @@ WHERE countersigned_agreement_path IS NOT NULL;
 -- Removes all personal data while keeping our app working as expected
 \echo 'Blank out declarations'
 UPDATE supplier_frameworks
-SET declaration = (
-    CASE
-    WHEN (declaration->'status') IS NULL
-    THEN '{}'
-    ELSE '{
-         "status": "' || (declaration->>'status') || '",
-         "primaryContactEmail": "supplier-user@example.com"
-         }'
-    END)::json
+SET
+    declaration = CASE WHEN (declaration->'status') IS NULL  -- i.e. key does not exist
+        THEN
+            '{}'::json
+        ELSE
+            json_build_object(
+                'status', (declaration->'status'),
+                'primaryContactEmail', 'supplier-user@example.com'
+            )
+    END
 WHERE declaration IS NOT NULL
-  AND declaration::varchar != 'null'
-  AND declaration::varchar != '{}';
+  AND json_typeof(declaration) != 'null'
+  AND declaration::text != '{}';
 
 -- Create some fake draft and submitted brief responses on closed briefs using eligible suppliers
```

Comment:
Running this locally I noticed this fake brief responses bit doesn't have an `echo` line for the output, hopefully it's not too out of scope to add one in 😸